### PR TITLE
Update Utility task for load test to show proper error message for soft shutdown

### PR DESCRIPTION
**Pull request by anshii03**
We want the CLT tasks to fail gracefully with proper error message indicating that the service is now shutdown.

**Task name**:  Cloud Load Test Task

**Description**: Catching proper error message and displaying so that customer knows that service is now shutdown.

**Documentation changes required:** N

**Added unit tests:** (Y/N)  N

**Attached related issue:** (Y/N) [https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/1831804]

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

Logs for cltabc account where cltshutdown is enabled :
image

Logs for clttestsu0 account where cltshutdown is disabled (similar behaviour as before)
image

### CODE REVIEW

File: Tasks/RunLoadTestV1/CltTasksUtility.ps1 [hunk 1]
```diff
@@ -2,7 +2,18 @@ function InvokeRestMethod($headers, $contentType, $uri , $method= "Get", $body)
 {
     $restTimeout = 60
     $ServicePoint = [System.Net.ServicePointManager]::FindServicePoint($uri)
-    $result = Invoke-RestMethod -ContentType "application/json" -UserAgent $global:userAgent -TimeoutSec $restTimeout -Uri $uri -Method $method -Headers $headers -Body $body
+     try
+	{
```
Comment (gnsip) [inline Tasks/RunLoadTestV1/CltTasksUtility.ps1:6]: Nit: proper indentation needed here

File: Tasks/RunLoadTestV1/CltTasksUtility.ps1 [hunk 2]
```diff
@@ -2,7 +2,18 @@ function InvokeRestMethod($headers, $contentType, $uri , $method= "Get", $body)
...
+   	 $result = Invoke-RestMethod -ContentType "application/json" -UserAgent $global:userAgent -TimeoutSec $restTimeout -Uri $uri -Method $method -Headers $headers -Body $body
+    }
+    catch
+	{
```
Comment (gnsip) [inline Tasks/RunLoadTestV1/CltTasksUtility.ps1:10]: Nit: proper indentation needed here

File: Tasks/RunLoadTestV1/CltTasksUtility.ps1 [hunk 3]
```diff
@@ -2,7 +2,18 @@ function InvokeRestMethod($headers, $contentType, $uri , $method= "Get", $body)
...
+    	Write-Host "An error occurred:"
+    	$errorObject = ConvertFrom-Json -InputObject $_.ErrorDetails
+    	$errorMessage = $errorObject.message
+    	Write-Error $errorMessage
+        throw "Error"
```
Comment (gnsip) [inline Tasks/RunLoadTestV1/CltTasksUtility.ps1:15]: Can you please be more clear with this message here? 
e.g. "Error occurred while calling API: $method $uri"

File: Tasks/RunLoadTestV1/CltTasksUtility.ps1 [hunk 4]
```diff
@@ -2,7 +2,18 @@ function InvokeRestMethod($headers, $contentType, $uri , $method= "Get", $body)
...
+    	}
     $ServicePoint.CloseConnectionGroup("")
     return $result
 }
```
Comment (gnsip) [inline Tasks/RunLoadTestV1/CltTasksUtility.ps1:19]: You need to update patch version of the task, please see the earlier commits for this task or any other task

**Review by ganesp — CHANGES_REQUESTED** [review]
Please address the review comments.
Also validate and share screenshots for both scenarios -- shutdown enabled/disabled

**Review by gnsip — APPROVED** [review]

**Review by gnsip — APPROVED** [review]

**Review by ganesp — APPROVED** [review]

**Pull request merged**